src/components.d.ts

```diff
@@ -72,6 +72,7 @@ export namespace Components {
     }
     interface SmoothlyDisplayAmount {
         "amount": number | string;
+        "concat": boolean;
```

Comment:
I believe `concat` is a shortening for `concatenate` so it should probably be renamed to that. But maybe another name would be a better fit? Maybe `integer`, `asInteger`or `tryInteger` would be a better fit?